pkg/sql/schema_changer.go

```diff
@@ -1601,9 +1601,19 @@ func (r schemaChangeResumer) Resume(
 				return r.job.MakeSessionBoundInternalExecutor(ctx, sd)
 			},
 		}
-		if err := sc.exec(ctx); err != nil {
+		opts := retry.Options{
+			InitialBackoff: 100 * time.Millisecond,
+			MaxBackoff:     20 * time.Second,
+			Multiplier:     1.5,
+		}
+		var scErr error
```

Comment:
I'm not sure it makes sense to retain this across iterations of the loop.

Reply:
no but we need it after we exit the loop to return to registry the last error from the schema change

Reply:
Makes sense.